PowerShell/ScubaGear/Modules/Providers/ProviderHelpers/SPOSiteHelper.psm1

```diff
@@ -9,10 +9,12 @@ function Get-SPOSiteHelper {
     param(
         [Parameter(Mandatory = $true, ParameterSetName = 'Report')]
         [ValidateSet("commercial", "gcc", "gcchigh", "dod")]
+        [ValidateNotNullOrEmpty()]
```

Comment:
ValidateSet is sufficient. Or course the additional ValidateNotNullOrEmpty will not hurt anything.